test/built-ins/DataView/prototype/getBigInt64/detached-buffer.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2017 Robin Templeton. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-dataview.prototype.getbigint64
+description: >
+  Throws a TypeError if buffer is detached
+info: |
+  DataView.prototype.getBigInt64 ( byteOffset [ , littleEndian ] )
+
+  1. Let v be the this value.
+  2. If littleEndian is not present, let littleEndian be undefined.
+  3. Return ? GetViewValue(v, byteOffset, littleEndian, "Int64").
+
+  24.3.1.1 GetViewValue ( view, requestIndex, isLittleEndian, type )
+
+  ...
+  8. Let buffer be the value of view's [[ViewedArrayBuffer]] internal slot.
+  9. If IsDetachedBuffer(buffer) is true, throw a TypeError exception.
+  ...
+includes: [detachArrayBuffer.js]
+features: [DataView, ArrayBuffer, BigInt, arrow-function]
+---*/
+
+var buffer = new ArrayBuffer(1);
+var sample = new DataView(buffer, 0);
+
+$DETACHBUFFER(buffer);
+assert.throws(TypeError, () => sample.getBigInt64(0), "0");
```

Comment:
Not sure why the message is "0", it could be something more descriptive, like "detached DataView access should throw".